src/nvim/api/command.c

```diff
@@ -598,6 +600,7 @@ String nvim_cmd(uint64_t channel_id, Dict(cmd) *cmd, Dict(cmd_opts) *opts, Error
 
     OBJ_TO_CMOD_FLAG(CMOD_SILENT, mods.silent, false, "'mods.silent'");
     OBJ_TO_CMOD_FLAG(CMOD_ERRSILENT, mods.emsg_silent, false, "'mods.emsg_silent'");
+    OBJ_TO_CMOD_FLAG(CMOD_UNSILENT, mods.silent, false, "'mods.unsilent'");
```

Comment:
for `nvim_cmd` we should probably give an error if `silent==unsilent` ?

Reply:
But have both `unsilent` and `silent` in a command is not an error, and there is even an example in `:h unsilent`:
```
		:silent argdo unsilent echo expand('%') .. ": " .. getline(1)
```

Reply:
`CMOD_UNSILENT` overrides `CMOD_SILENT`, and there is actually a plugin that uses both `:silent!` and `:unsilent`: <https://github.com/Shougo/neobundle.vim/blob/1306c131ff0a6cc094b7ff0727350e442b5427f1/autoload/neobundle/config.vim#L665>

`:silent!` suppresses error messages, while `:unsilent` enables normal messages. And `:silent!` sets both `CMOD_ERRILENT` and `CMOD_SILENT`, so if a dictionary passed to `nvim_cmd` comes from `nvim_parse_cmd` or Lua user command it may have `emsg_silent`, `silent` and `unsilent` all set to `true`.